odoo/addons/base/security/ir.model.access.csv

```diff
@@ -95,7 +87,7 @@
 "access_ir_filter_public","ir_filters all","model_ir_filters","group_public",1,1,1,1
 "access_ir_config_parameter_system","ir_config_parameter_system","model_ir_config_parameter","group_system",1,1,1,1
 "access_ir_mail_server","ir_mail_server","model_ir_mail_server","group_system",1,1,1,1
-"access_ir_actions_client","ir_actions_client all","model_ir_actions_client",,1,0,0,0
+"access_ir_actions_client","ir_actions_client all","model_ir_actions_client","group_system",1,0,0,0
```

Comment:
bit odd that group_system can update & delete all other action types but can only see client actions no? Though there's no menu for client actions in basem maybe this rule should be moved next to the other action rules, and also made fully CRUD?